x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts

```diff
@@ -146,9 +159,8 @@ export async function getServiceTransactionGroups({
       ...transactionGroup,
       transactionType,
     })),
-    isAggregationAccurate:
-      (response.aggregations?.transaction_groups.sum_other_doc_count ?? 0) ===
-      0,
-    bucketSize,
+    maxTransactionGroupsExceeded:
+      (response.aggregations?.transaction_groups.sum_other_doc_count ?? 0) > 0,
+    overflowCount: response.aggregations?.overflow_count.value ?? 0,
```

Comment:
Let's call this `transactionOverflowCount` to distinguish from `serviceOverflowCount`:

```suggestion
    transactionOverflowCount: response.aggregations?.overflow_count.value ?? 0,
```